iquip/apps/dataviewer.py

```diff
@@ -4,18 +4,23 @@
 import dataclasses
 import enum
 import logging
-from typing import Sequence, Optional
+from typing import Dict, Sequence, Optional, Union
 
 import numpy as np
 import pyqtgraph as pg
 from PyQt5.QtWidgets import (
     QWidget, QLabel, QPushButton, QRadioButton, QButtonGroup, QStackedWidget,
-    QAbstractSpinBox, QSpinBox, QHBoxLayout, QVBoxLayout
+    QAbstractSpinBox, QSpinBox, QDoubleSpinBox,
+    QHBoxLayout, QVBoxLayout, QGridLayout,
 )
+from PyQt5.QtCore import pyqtSignal, pyqtSlot
 
 logger = logging.getLogger(__name__)
 
 
+MAX_INT = 2**31 - 1
```

Comment:
How about removing an empty line before this statement?

We usually add only one line between defining global variables or constants.
Please refer to [test_builder.py](https://github.com/snu-quiqcl/iquip/blob/develop/tests/test_builder.py).